tls/s2n_config.h

```diff
@@ -50,6 +50,11 @@ struct s2n_config {
     s2n_cert_auth_type client_cert_auth_type;
     verify_cert_trust_chain_fn *verify_cert_chain_cb;
     void *verify_cert_context;
+
+    uint8_t mfl_code;
+
+    /* if this is FALSE, server will ignore client's Maximum Fragment Length request */
+    int enable_server_mfl;
```

Comment:
I think we can align these two variable names with the API calls, how about `mlf_code_to_send` and `accept_mfl`?